R/bed_cluster.r

```diff
@@ -1,20 +1,21 @@
 #' Cluster neighboring intervals.
-#' 
+#'
 #' Output contains an \code{.id} column that can be used in downstream grouping
 #' operations. Default \code{max_dist = 0} means that both overlapping and
-#' book-ended intervals will be clustered.
-#' 
-#' @param x tbl of intervals
+#' book-ended intervals will be clustered. Note that input tbl is assumed to be sorted
```

Comment:
I would prefer to just sort this in the bed_cluster function itself, rather than requiring  explicit sort operation (one less thing to remember to do).